packages/dracut/sysext/files/90kairos-sysext/module-setup.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# This module tries to add systemd-sysext module to the initramfs if its in the system, otherwise it does nothing
+
+# called by dracut
+check() {
+    return 0
+}
+
+# called by dracut
+depends() {
+    # If the binary(s) requirements are not fulfilled the module can't be installed.
+    require_binaries systemd-sysext || return 1
+    # Check if the module files exists
+    # This is not normal but on ubuntu-22 the binary for sysext exists but the dracut module doesnt, so we
+    # need to do further checks before including it otherwise it will fail to build the initrd
+    files=( "${dracutbasedir}"/modules.d/??systemd-sysext )
+    [ "${#files[@]}" -ge 2 ] && return 1
+    if [ -d "${files[0]}" ]; then
+        echo "systemd-sysext"
```

Comment:
this is what brings the systemd-sysext module as dependent, so it includes on systems that have botht he binary and the dracut module available, otherwise this module depends on nothing and its basically an empty thing that does nothing.